apps/console/src/extensions/configs/models/common.ts

```diff
@@ -32,7 +32,8 @@ export interface CommonConfig {
          * Get the extensions for the header.
          * @returns Header extensions.
          */
-        getHeaderExtensions: () => Promise<HeaderExtension[]>;
+        getHeaderExtensions: (tenantDomain: string,
```

Comment:
Shall we format this as suggested above?

Reply:
Fixed with b5a3098